src/Playwright.Tests/PageRouteTests.cs

```diff
@@ -691,5 +691,23 @@ public async Task ShouldSupportCorsWithDifferentMethods()
 
             Assert.AreEqual(new[] { "DELETE", "electric", "cars" }, resp);
         }
+
+        [PlaywrightTest("page-route.spec.ts", "should throw on invalid route url")]
```

Comment:
nit: as this test is specific to C#, you don't need to annotate it with the link to the upstream test.